.github/workflows/asana-release.yml

```diff
@@ -262,19 +354,40 @@ jobs:
           body: "${{ env.PR_BODY_WINDOWS }}"
           token: ${{ secrets.DAXMOBILE_AUTOFILL_AUTOMATION }}
 
+# ------------------------------------------------------------------------------
+# Update Asana subtasks with PR URLs
+# ------------------------------------------------------------------------------
+
+  update_asana_tasks:
+    runs-on: ubuntu-20.04
+    if: ${{ always() }} # Always run this final step, even if any of the updates have failed
+    needs: [create_asana_tasks, update_bsk, update_ios, update_macos, update_android, update_extensions, update_windows]
+    steps:
+      # --- Setup ---
+      - uses: actions/checkout@v3
+        with:
+          path: autofill/
+      - uses: ./autofill/.github/actions/setup-release-scripts
+      # --- Effect ---
       - name: Update Asana tasks
         env:
           ASANA_ACCESS_TOKEN: ${{ secrets.NATIVE_APPS_WORKFLOW }}
-          VERSION: ${{ github.event.release.tag_name }}
-          RELEASE_URL: ${{ github.event.release.html_url }}
-          RELEASE_NOTES: ${{ github.event.release.body }}
-          ASANA_OUTPUT: ${{ steps.create-asana-tasks.outputs.ASANA_OUTPUT }}
-          BSK_PR_URL: ${{ steps.bsk-pr.outputs.pull-request-url }}
-          IOS_PR_URL: ${{ steps.ios-pr.outputs.pull-request-url }}
-          MACOS_PR_URL: ${{ steps.macos-pr.outputs.pull-request-url }}
-          ANDROID_PR_URL: ${{ steps.android-pr.outputs.pull-request-url }}
-          WINDOWS_PR_URL: ${{ steps.windows-pr.outputs.pull-request-url }}
-        run:  node ./autofill/scripts/release/asana-update-tasks.js
+          ASANA_OUTPUT: ${{ needs.create_asana_tasks.outputs.asana-output }}
+          BSK_PR_URL: ${{ needs.update_bsk.outputs.pull-request-url }}
+          IOS_PR_URL: ${{ needs.update_ios.outputs.pull-request-url }}
+          MACOS_PR_URL: ${{ needs.update_macos.outputs.pull-request-url }}
+          ANDROID_PR_URL: ${{ needs.update_android.outputs.pull-request-url }}
+          WINDOWS_PR_URL: ${{ needs.update_windows.outputs.pull-request-url }}
+        run: |
+          node ./autofill/scripts/release/asana-update-tasks.js
```

Comment:
Does this still run if any of the PR URLs above aren't available?

Reply:
Update: Yes, this will still run if the PR URLs aren't available -- it defaults the value to `"error"`, which is then used as the URL for the link href. That should make it relatively easy to spot and update as part of any manual  intervention required.